objstorage/objstorageprovider/provider_test.go

```diff
@@ -593,7 +593,7 @@ func TestParallelSync(t *testing.T) {
 							}
 						}
 					}
-				}(numOps*n, shared)
+				}(numOps*(n+1), shared)
```

Comment:
This change is to ensure that we don't get a FileNum of 0 which will trigger various assertions during the invariant build.